content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -637,9 +637,9 @@ Get the Pods to view their container images:
 for p in 0 1 2; do kubectl get pod "web-$p" --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'; echo; done
 ```
 ```
-registry.k8s.io/nginx-slim:0.8
-registry.k8s.io/nginx-slim:0.8
-registry.k8s.io/nginx-slim:0.8
+gcr.io/google_containers/nginx-slim:0.8
+gcr.io/google_containers/nginx-slim:0.8
+gcr.io/google_containers/nginx-slim:0.8
```

Comment:
@sftim @tomoaki-ii 
I think the `gcr.io` is replaced with `registry.k8s.io` on the community.
This updates does not follow the rules, I think.

WDYT?

Reply:
At a same time, If we use `registry.k8s.io/nginx-slim:0.8` in the patch command [here](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#rolling-update), it does not create any changes in the manifest, right?

I'm confusing about the scenerio of this page. :thinking: